nixos/doc/manual/configuration/wayland.xml

```diff
@@ -20,4 +20,14 @@
   essential utilities. Now you can start <package>sway</package> from the TTY
   console.
  </para>
+
+ <para>
+  If you are using a wlroots-based compositor, like sway, and want to be able to
+  share your screen, you might want to activate this option:
```

Comment:
I want to activate this option, do I?

Reply:
I'm not sure I understand the comment. If this is about the wording "you might want to" instead of "you will want to", it's just that the end user can choose to do `xdg.portal.extraPortal = [ pkgs.xdg-desktop-portal-wlr ]` instead and it will work fine. It just won't have any configuration file.

Reply:
Not sure what you mean.